examples/atlas/create-cluster.md

```diff
@@ -0,0 +1,15 @@
+# Create a Cluster
+
+To create and get the connection string for a cluster you can do:
+
+```bash
+mongocli atlas clusters create MyCluster \
+  --region EASTERN_US \
+  --members 3 \
```

Comment:
[nit] the default value of `--members` is 3 so you can remove this

Reply:
I like to have examples showing it so people know it exists